### ISSUE TITLE
Unable to set multiple tags in bake cmd

### ISSUE THREAD

**Issue by zachary-povey** [docker/buildx#872]
## Steps to reproduce
- Create an empty dir
- Add a `Dockerfile`:
```
FROM centos:8
```
- Add a `docker-bake.hcl`:
```
group "default" {
  targets = ["image"]
}
target "image" {
  context = "."
}
```
- Run docker `buildx bake --set "*.tags=image_name_1" image --load` - this passes as expected and loads `image_1:latest` into the local docker instance
- Run `docker buildx bake --set "*.tags=image_name_1,image_name_2" image --load` - this failes with:
```
[+] Building 0.0s (0/0)                                                                                                                                              
error: invalid tag "image_name_1,image_name_2": invalid reference format
```

## Expected behaviour
Based on the plural name, and looking at the code, I believe the intent is for `target.tags` to be an array - am I using the wrong separator or is this a bug?

**Comment by marcellodesales** [comment]
I particularly had a hard time to get around it as well... It would be great if this is well documented with examples...

**Comment by pongnguy** [comment]
Similar comment regarding docker buildx bake -f, where -f takes a stringArray.  I want to pass in multiple docker-bake.json files.

**Comment by berney** [comment]
In the a docker-bake.hcl file, we would set the tags like this:

```hcl
# docker-bake.hcl
target "webapp-dev" {
  dockerfile = "Dockerfile.webapp"
  tags = ["docker.io/username/webapp:latest"]
}
```

Where `tags` takes an array. So, I expected the arg to `--set` to accept an array, e.g. `docker buildx bake --set 'webapp-dev.tags=[foo/bar:latest,foo/bar:v1.2.3]'`, that didn't work and searching I found this issue.

I would have expected repeating  args, that the last arg would override the previous args. I think it makes more sense that the arg is an array, like the HCL syntax.

**Comment by neersighted** [comment]
It turns out shell brace expansion is a pretty good fit here, so maybe we can document that? e.g. `docker buildx bake all --set=\*.tags=foo/bar:{baz,qux}`.

**Comment by tsouza-squid** [comment]
Not sure if documentation was updated recently, but this is buried down in the [docs](https://docs.docker.com/reference/cli/docker/buildx/bake/#set):
```bash
docker buildx bake --set target.platform+=linux/arm64 # appends 'linux/arm64' to the platform list
```

**Comment by crazy-max** [comment]
> Not sure if documentation was updated recently, but this is buried down in the [docs](https://docs.docker.com/reference/cli/docker/buildx/bake/#set):
> 
> docker buildx bake --set target.platform+=linux/arm64 # appends 'linux/arm64' to the platform list

Yes you can append with `+=` operator since 0.22.0: https://docs.docker.com/build/release-notes/#0220

```
$ docker buildx bake --set "*.tags+=image_name_1" --set "*.tags+=image_name_2" --load image --print
{
  "group": {
    "default": {
      "targets": [
        "image"
      ]
    }
  },
  "target": {
    "image": {
      "context": ".",
      "dockerfile": "Dockerfile",
      "tags": [
        "image_name_1",
        "image_name_2"
      ],
      "output": [
        {
          "type": "docker"
        }
      ]
    }
  }
}
```

This is useful if you have tags already defined in your bake definition, otherwise you can just do:

```
$ docker buildx bake --set "*.tags=image_name_1" --set "*.tags=image_name_2" --load image --print
{
  "group": {
    "default": {
      "targets": [
        "image"
      ]
    }
  },
  "target": {
    "image": {
      "context": ".",
      "dockerfile": "Dockerfile",
      "tags": [
        "image_name_1",
        "image_name_2"
      ],
      "output": [
        {
          "type": "docker"
        }
      ]
    }
  }
}
```

Multiple flags are supported for `--set`: https://github.com/spf13/pflag/blob/d5e0c0615acee7028e1e2740a11102313be88de1/string_array.go#L75-L77

And agree that we should have an example for it:
* https://docs.docker.com/reference/cli/docker/buildx/bake/#set
* https://docs.docker.com/build/bake/overrides/
